README.md

```diff
@@ -25,6 +25,11 @@ You can contribute interesting links that might be helpful for our community, fo
 - [Getting Started and Prerequisites](https://discussions.udacity.com/t/getting-started-and-prerequisites/418485?u=haitec) (Udacity Discussion Forum)
 - [Video: Jake Archibald about progressive apps](https://www.youtube.com/watch?v=cmGr0RszHc8) (from Google I/O 2016)
 
+
+### Upon Completion
```

Comment:
This might be a misleading title.

Reply:
Source: https://web.archive.org/web/20171003221949/https://udacity.com/course/mobile-web-specialist-nanodegree--nd024